CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.10)
+cmake_minimum_required (VERSION 3.5)
 project(parakeet-qt VERSION 1.0.2 LANGUAGES CXX)
```

Comment:
Seems like we forgot to bump the version here

Reply:
Good catch. Fixed